app/stories/pages/how-it-works.stories.tsx

```diff
@@ -2,11 +2,23 @@ import { ComponentMeta, ComponentStory } from '@storybook/react'
 
 import HowItWorksPage from '@pages/how-it-works'
 
+import { getBackRoute, getForwardRoute } from '@utils/routing'
+
+import { getEmptyMockSession } from '../../tests/helpers/mockData'
+
 export default {
   title: 'Pages/How It Works',
   component: HowItWorksPage,
 } as ComponentMeta<typeof HowItWorksPage>
 
-const Template: ComponentStory<typeof HowItWorksPage> = () => <HowItWorksPage />
+const Template: ComponentStory<typeof HowItWorksPage> = (args) => {
+  args.backRoute = getBackRoute('/how-it-works', getEmptyMockSession())
```

Comment:
Notes for reviewers: Similar to `forwardRoute`, in the storybook story, we mock `backRoute` by calling `getBackRoute()` directly. In the application, `_app.tsx` would handle making this call. `getBackRoute()` has the following function signature:

```ts
function getBackRoute(
  pathname: string,
  session: SessionData | ((value: SessionData) => void)
)
```